src/basic_operators.jl

```diff
@@ -0,0 +1,94 @@
+struct DiffEqIdentity{T,N} <: AbstractDiffEqLinearOperator{T} end
+DiffEqIdentity(u) = DiffEqIdentity{eltype(u),length(u)}()
```

Comment:
Will we need this in DiffEqBase to be the default mass matrix?

Reply:
I think so.